services/localzone.c

```diff
@@ -1638,7 +1685,8 @@ local_zones_zone_answer(struct local_zone* z, struct module_env* env,
 	} else if(lz_type == local_zone_refuse
 		|| lz_type == local_zone_always_refuse) {
 		local_error_encode(qinfo, env, edns, repinfo, buf, temp,
-			LDNS_RCODE_REFUSED, (LDNS_RCODE_REFUSED|BIT_AA));
+			LDNS_RCODE_REFUSED, (LDNS_RCODE_REFUSED|BIT_AA),
+			LDNS_EDE_BLOCKED, "");
```

Comment:
Part of the not so simple EDE cases?

```suggestion
			LDNS_RCODE_REFUSED, (LDNS_RCODE_REFUSED|BIT_AA), -1, NULL);
```